src/frontend/js/widgets/Dashboard/components/TeacherDashboardProfileSidebar/components/OrganizationLinks/_styles.scss

```diff
@@ -33,9 +34,9 @@
         box-shadow: 0 0 rem-calc(3px) r-theme-val(organization-thumb, border-color);
       }
       &__img {
-        object-fit: cover;
+        object-fit: contain;
         object-position: center center;
-        aspect-ratio: 2/1;
+        aspect-ratio: 1/1;
```

Comment:
curious: can't you just set `aspect-ratio: 1`?

Reply:
Yup it should work but from my point of view it's unclear. A ratio implies the division of two values.